scripts/src/checkprcontent/checkpr.py

```diff
@@ -54,7 +54,7 @@ def check_provider_delivery(report_in_pr,num_files_in_pr,report_file_match):
     provider_delivery = False
     if report_in_pr and num_files_in_pr > 1:
         if report_provider_delivery or owner_provider_delivery:
-            msg = f"[ERROR] OWNERS file and/or report indicate provider controlled delivery but pull request is not report only."
+            msg = f"[ERROR] OWNERS file and/or report indicate web catalog only delivery but pull request is not report only. The distribution method web catalog only requires providerDelivery to be set to true in the OWNERS file."
```

Comment:
I wanted to get away from referring the "OWNERS file" and instead "the distribution method selected"

Reply:
So something like "Web catalog only requires providerDelivery to be set to true in the distribution method selected"?